apollo-client/src/test/java/com/ctrip/framework/apollo/BaseIntegrationTest.java

```diff
@@ -108,10 +120,22 @@ public void setUp() throws Exception {
     ReflectionTestUtils.invokeMethod(MetaDomainConsts.class, "reset");
 
     MockInjector.setInstance(ConfigUtil.class, new MockConfigUtil());
+
+    configDir = new File(ClassLoaderUtil.getClassPath() + "config-cache");
+    if (configDir.exists()) {
+      configDir.delete();
+    }
+    configDir.mkdirs();
   }
 
   @AfterEach
   public void tearDown() throws Exception {
+    // get the instance will trigger long poll task execute, so move it from setup to tearDown
+    RemoteConfigLongPollService remoteConfigLongPollService
+        = ApolloInjector.getInstance(RemoteConfigLongPollService.class);
```

Comment:
`ApolloInjector.getInstance(RemoteConfigLongPollService.class);` will trigger context load, so move it to `tearDown` method